src/SpecFlow.Dnx/Fixer.cs

```diff
@@ -1,238 +1,269 @@
-
-			var featureFiles = directory.GetFiles("*.feature", SearchOption.AllDirectories);
-			var sb = new StringBuilder();
-
-			// Set the "ToolsVersion" to VS2013, see: https://github.com/techtalk/SpecFlow/issues/471
-			sb.Append(@"<?xml version=""1.0"" encoding=""utf-8""?>
-<Project ToolsVersion=""14.0"" DefaultTargets=""Build"" xmlns=""http://schemas.microsoft.com/developer/msbuild/2003"">
-	<PropertyGroup>
-		<RootNamespace>SpecFlow.GeneratedTests</RootNamespace>
-		<AssemblyName>SpecFlow.GeneratedTests</AssemblyName>
-	</PropertyGroup>
-	<ItemGroup>
-		<None Include=""app.config"">
-			<SubType>Designer</SubType>
-		</None>");
-
-			foreach (var featureFile in featureFiles)
-			{
-				sb.Append($@"
-		<None Include=""{featureFile.FullName.Replace(directory.FullName + Path.DirectorySeparatorChar, "")}"">
-			<Generator>SpecFlowSingleFileGenerator</Generator>
-			<LastGenOutput>{featureFile.Name}.cs</LastGenOutput>
-		</None>");
-			}
-
-			sb.Append(@"
-	</ItemGroup>
-</Project>");
-
-			var content = sb.ToString();
-			Console.WriteLine(content);
-
-			var csprojPath = xproj.FullName + ".fake.csproj";
-			Console.WriteLine("Saving: " + csprojPath);
-			File.WriteAllText(csprojPath, content);
-
-			return new FileInfo(csprojPath);
-		}
-
-		private FileInfo GetXproj(DirectoryInfo directory)
-		{
-			var xprojs = directory.GetFiles("*.xproj", SearchOption.TopDirectoryOnly);
-
-			if (xprojs.Length == 0)
-				throw new Exception("Could not find '.xproj'.");
-
-			if (xprojs.Length > 1)
-				throw new Exception("More than one '.xproj' found.");
-
-			var xproj = xprojs.Single();
-			Console.WriteLine("Found: " + xproj.FullName);
-
-			return xproj;
-		}
-	}
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Xml.Linq;
+using System.Xml.XPath;
+
+namespace SpecFlow.Dnx
+{
+    internal class Fixer
+    {
+        public readonly string SpecFlowExe;
+
+        private const string AppConfigSpecFlowSectionDefinitionType = "TechTalk.SpecFlow.Configuration.ConfigurationSectionHandler, TechTalk.SpecFlow";
+
+        private const string AppConfigSpecFlowSectionDefinition = @"	<configSections>
+		<section name=""specFlow"" type=""" + AppConfigSpecFlowSectionDefinitionType + @""" />
+	</configSections>";
+
+        private const string AppConfigSpecFlowSectionElement = "unitTestProvider";
+
+        // TODO: Allow specifying other unit test providers.
+        private const string AppConfigSpecFlowSection = @"	<specFlow>
+		<" + AppConfigSpecFlowSectionElement + @" name=""xUnit"" />
+	</specFlow>";
+
+        public Fixer()
+        {
+            // TODO: Allow for other DNX install locations.
+            var users = Directory.GetDirectories(@"C:\Users");
+
+            var dnxUser = users.SingleOrDefault(u =>
+            {
+                try
+                {
+                    return Directory.GetDirectories(u).Any(d => d.Contains(".dnx"));
```

Comment:
I doubt we can rely on this folder for people new to .NET Core (i.e. who don't have legacy DNX folders hanging around).